internal/cmd/postgresflex/instance/describe/describe.go

```diff
@@ -0,0 +1,128 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex"
+)
+
+const (
+	instanceIdArg = "INSTANCE_ID"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	InstanceId string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", instanceIdArg),
+		Short: "Get details of a PostgreSQL Flex instance",
+		Long:  "Get details of a PostgreSQL Flex instance.",
+		Args:  args.SingleArg(instanceIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a PostgreSQL Flex instance with ID "xxx"`,
+				"$ stackit postgresflex instance describe xxx"),
+			examples.NewExample(
+				`Get details of a PostgreSQL Flex instance with ID "xxx" in a table format`,
+				"$ stackit postgresflex instance describe xxx --output-format pretty"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read PostgreSQL Flex instance: %w", err)
+			}
+
+			return outputResult(cmd, model.OutputFormat, resp.Item)
+		},
+	}
+	return cmd
+}
+
+func parseInput(cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	instanceId := inputArgs[0]
+
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		InstanceId:      instanceId,
+	}, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *postgresflex.APIClient) postgresflex.ApiGetInstanceRequest {
+	req := apiClient.GetInstance(ctx, model.ProjectId, model.InstanceId)
+	return req
+}
+
+func outputResult(cmd *cobra.Command, outputFormat string, instance *postgresflex.Instance) error {
+	switch outputFormat {
+	case globalflags.PrettyOutputFormat:
+		acls := *instance.Acl.Items
+		strings.Join(acls, ",")
```

Comment:
don't you need to assign this to the `acls` variable? In the `mongodbflex` it should also be fixed

Reply:
Good catch!